### DISCUSSION TITLE
We need 1.18.2

### DISCUSSION THREAD

**Discussion by CreeperPeruano** [Draylar/identity#350] [General]
So, I'll be short. We need 1.18.2 pleaseeeee! Seriously, this has been a cool mod and the only alternative to the old morph back in its days, so it would be cool to see a 1.18.2 version :)

**Comment by PsychoBreadstick** [comment]
Yep, I agree. I'm surprised there isn't a 1.18.2 version already.

**Comment by Draylar** [comment]
1.18.2 is out now! :)
